src/models/Project.js

```diff
@@ -1,9 +1,9 @@
 const Project =
     'CREATE TABLE IF NOT EXISTS "Project" (\
         id SERIAL,\
-        name VARCHAR(255) NOT NULL, \
+        name CITEXT NOT NULL, \
```

Comment:
Wouldn't it be better if we could make the project name case-sensitive? I think you're going for the GH look, but besides Git platforms, project names are typically case-sensitive. It also provides an aesthetic that makes the app look more "real", functional", and professional.

Reply:
The names are still stored as case-sensitive values and are only treated as case-insensitive for comparison purposes, so when they're retrieved from the database, they'll still retain their original case.

Reply:
Amazing!